### PR TITLE
Add catalog.cattle.io/os linux annotation to rancher-externai-ip-webhook

### PR DESCRIPTION
**Pull request by rawmind0**

### CODE REVIEW

File: packages/rancher-external-ip-webhook/package.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/externalip-webhook/releases/download/v0.1.5/rancher-externalip-webhook-0.1.5.tgz
-packageVersion: 00
+url: https://github.com/rancher/externalip-webhook/releases/download/v0.1.6/rancher-externalip-webhook-0.1.6.tgz
+packageVersion: 01
```
Comment (cbron) [inline packages/rancher-external-ip-webhook/package.yaml:2]: 00 is fine ?

File: packages/rancher-external-ip-webhook/rancher-external-ip-webhook.patch [hunk 2]
```diff
@@ -1,40 +1,28 @@
...
    catalog.cattle.io/display-name: External IP Webhook
    catalog.cattle.io/namespace: cattle-externalip-system
+   catalog.cattle.io/os: linux
 -  catalog.cattle.io/release-name: rancher-externalip-webhook
 -  catalog.cattle.io/ui-component: rancher-externalip-webhook
 +  catalog.cattle.io/release-name: rancher-external-ip-webhook
 +  catalog.cattle.io/ui-component: rancher-external-ip-webhook
  apiVersion: v1
- appVersion: v0.1.5
+ appVersion: v0.1.6
```
Comment (cbron) [inline packages/rancher-external-ip-webhook/rancher-external-ip-webhook.patch:13]: I don't see why this is necessary if we are rebasing to 0.1.6  in the package.yaml

Comment (rawmind0) [reply packages/rancher-external-ip-webhook/rancher-external-ip-webhook.patch:13]: We don't set the appVersion??

File: packages/rancher-external-ip-webhook/package.yaml (hunk 1 above)
Comment (rawmind0) [reply packages/rancher-external-ip-webhook/package.yaml]: updated

**Review by cbron — APPROVED** [review]

**Pull request merged**